be/src/olap/task/engine_storage_migration_task.cpp

```diff
@@ -341,19 +290,10 @@ void EngineStorageMigrationTask::_generate_new_header(
 
 Status EngineStorageMigrationTask::_copy_index_and_data_files(
         const string& full_path, const std::vector<RowsetSharedPtr>& consistent_rowsets) const {
-    Status status = Status::OK();
     for (const auto& rs : consistent_rowsets) {
-        status = rs->copy_files_to(full_path, rs->rowset_id());
-        if (!status.ok()) {
-            Status ret = FileUtils::remove_all(full_path);
```

Comment:
This line is removed, then there exists garbage in this folder?

Reply:
`EngineStorageMigrationTask::_migrate` has done this garbage collection.